### PR TITLE
chore(proxy): obliterate nuke binary

### LINKED ISSUE #932: Move the nuke binary functionality into proxy
This would simplify my life using it for QA. This way we won't have to package another binary.
I also propose to rename `nuke` to something more friendly, like `--factory-reset`.

I'd expect these to be equivalent: `./nuke` === `./proxy --factory-reset`.

### PR DESCRIPTION
**Pull request by xla**
Fix the aggressive wording and offer a command on the main proxy binary
`--factory-reset` which resets all local state.

Closes #932

_Be aware this is on top of `rudolfs/qa-checklist` to avoid merge conflicts and aid that PR._

### CODE REVIEW

**Review by NunoAlexandre — COMMENTED** [review]
How do others feel about "factory resetting"? It makes it sound as if we have some sort of "factory default" state when in fact we are just deleting the state. no biggie, especially if it doesn't confuse anyone.

**Comment by sarahscott** [comment]
Yeah the notion of a "factory" is nostalgic - maybe something like `clean` or `clearState` or `resetState` makes sense. Agree that it's by no means a blocking issue

File: QA.md [hunk 1]
```diff
@@ -19,10 +19,10 @@
       look like this:
 
   ```
-  INFO  nuke > Nuking application state...
-  INFO  nuke > done
-  INFO  nuke > Nuking coco state...
-  INFO  nuke > done
+  INFO  api > Resetting application state...
```
Comment (sarahscott) [inline QA.md:22]: 👍

**Comment by NunoAlexandre** [comment]
Yeah, "clear state" is a clean cut :)

**Comment by rudolfs** [comment]
> Yeah, "clear state" is a clean cut :)

Sounds good to me too.

**Comment by xla** [comment]
Renamed based on input from @NunoAlexandre and @sarahscott.

**Comment by xla** [comment]
@rudolfs If you feel comfortable with the changes, feel free to merge into your branch whenever is convenient.

**Review by NunoAlexandre — APPROVED** [review]

File: proxy/coco/src/control.rs [hunk 2]
```diff
@@ -10,12 +10,12 @@ use radicle_surf::vcs::git::git2;
...
 
-/// Deletes the local git repsoitory coco uses to keep its state.
+/// Deletes the local git repsoitory coco uses for its state.
 ///
 /// # Errors
 ///
-/// Will error in case the call to the [`std::fs::remove_dir_all`] fails.
-pub fn nuke_monorepo() -> Result<(), std::io::Error> {
+/// * if the call to [`std::fs::remove_dir_all`] fails.
+pub fn wipe_monorepo() -> Result<(), std::io::Error> {
```
Comment (FintanH) [inline proxy/coco/src/control.rs:18]: Why is _this_ one called wipe and not reset? :sweat_smile:

Comment (rudolfs) [reply proxy/coco/src/control.rs:18]: Yeah, was also wondering. @xla?

I could still rename it in my branch...

Comment (xla) [reply proxy/coco/src/control.rs:18]: :shrug: - creative freedom.

Comment (FintanH) [reply proxy/coco/src/control.rs:18]: :art:

Comment (xla) [reply proxy/coco/src/control.rs:18]: freedom

**Pull request merged**